### PR TITLE
 [AP-3573] certificate page for bank statement upload

### PR DESCRIPTION
**Pull request by jsugarman**
## What
Update certificate page for (enhanced) bank statement upload applications

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3573)

This addresses the following
- [X] Add feature tests to cover regressions
- [x] The enhanced bank statement journey requires changes inline with designs
   namely, replace “Income, regular payments and assets” section with equivalent CYA/means-summary sections.
- [X] ~~Prevent display of housing benefit total in income for truelayer application~~ separate PR merged
- [X] ~~Fix student loan section - displays but with empty amount on truelayer applications that do no add it.~~ separate ticket

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en/shared.yml [hunk 1]
```diff
@@ -223,6 +223,9 @@ en:
         request_higher_limit: Higher cost limit requested
         new_cost_limit: Amount requested
         new_limit_reasons: Reason
+      housing_benefit:
+        heading: Housing Benefit
+        does_your_client: Does your client receive housing benefits?
```
Comment (jsugarman) [inline config/locales/en/shared.yml:228]:
```suggestion
        does_your_client: Does your client receive Housing Benefit?
```

File: features/providers/review_and_print.feature [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      | h2  | Variation or discharge under section 5 protection from harassment act 1997r |
+      | h2  | Extend, variation or discharge - Part IV |
+      | h1  | Print your application |
+
+  Scenario: For a non-passported enhanced bank statement upload journey
+    Given the feature flag for enhanced_bank_upload is enabled
+    And I have completed an enhanced bank statement upload application with merits
+    When I view the review and print your application page
+
+And I save and open page
```
Comment (jsugarman) [inline features/providers/review_and_print.feature]: remove

**Review by RoseSAK — APPROVED** [review]
All looks good except for one small comment 🎉

File: app/views/shared/review_application/_income_payments_and_assets_for_bank_statement_upload.html.erb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        read_only: true
+      ) %>
+  </section>
+
+  <section class="print-no-break">
+    <%= render('shared/check_answers/housing_benefit', read_only: true) if @legal_aid_application.housing_benefit_regular_transaction_applicable? %>
+  </section>
+
+  <section class="print-no-break">
+    <%= render('shared/check_answers/income_cash_payments', read_only: true) if @legal_aid_application.uploading_bank_statements? %>
```
[the remark this answers is not in the document]
Comment (jsugarman) [reply app/views/shared/review_application/_income_payments_and_assets_for_bank_statement_upload.html.erb:36]: ooh, yes

**Pull request merged**